src/main.cpp

```diff
@@ -424,8 +425,10 @@ static void setup_log_params(const Settings &cmd_args)
 			Logger::color_mode = LOG_COLOR_ALWAYS;
 		else if (color_mode == "never")
 			Logger::color_mode = LOG_COLOR_NEVER;
-		else
+		else{
```

Comment:
Space after `else`. Additionally, each block above would need braces due to the Linux kernel style guidelines we use in this project.